woocommerce-payments.php

```diff
@@ -106,7 +106,7 @@ function wcpay_init() {
 	 * Initialise subscriptions-core if WC Subscriptions (the plugin) isn't loaded
 	 */
 	function wcpay_init_subscriptions_core() {
-		if ( ! WC_Payments_Features::is_wcpay_subscriptions_enabled() ) {
+		if ( ! class_exists( 'WooCommerce' ) || ! WC_Payments_Features::is_wcpay_subscriptions_enabled() ) {
```

Comment:
`class_exists( 'WooCommerce' )` is returning `false` for me even with WooCommerce active 🤔 This results in subscriptions core not loading.

I wonder if we need to do something similar to line `101` and run `wcpay_init_subscriptions_core` on the `plugins_loaded` hook?

Reply:
Hey @aprea, I've just pushed up a commit which inits subscriptions core on the `plugins_loaded` hook but with priority `0`.

Why priority 0?

WCS Core already hooks onto [`plugins_loaded`](https://github.com/Automattic/woocommerce-subscriptions-core/blob/da32cf1d9073b40ca32d34a9381a9a6d2288aeb3/includes/class-wc-subscriptions-core-plugin.php#L141-L145) and other early hooks like `woocommerce_blocks_loaded` which I believe is hooked onto `plugins_loaded` 9, so we want to make sure Subscriptions Core is inited as early as possible.

I've just gone through the list of action hooks in WCS Core to make sure nothing would be already called/missed now that we're initing on plugins loaded 0 and I couldn't see anything.

Because this is a fairly big change to subscriptions, I also performed some quick smoke tests:
 - Create subscriptions product
 - Purchase a subscription on checkout
 - Test checkout blocks with subscriptions
 - Cancel a subscription
 - Update subscription settings
 - Disable/enable WCPay Subscriptions feature

All were fine ✅ 

Other tests:
 - WooCommerce deactivated 👍 
 - WooCommerce Admin activated + deactive Woocommerce 👍 
 - WooCommerce activated :+1: